diff -up nvidia-settings-1.0/Makefile.default nvidia-settings-1.0/Makefile
--- nvidia-settings-1.0/Makefile.default	2008-06-06 21:25:57.000000000 +0200
+++ nvidia-settings-1.0/Makefile	2008-06-06 21:30:15.000000000 +0200
@@ -102,10 +102,10 @@ LD_RUN_FLAG=-R/usr/X11R6/lib
 else
 ECHO=echo
 define INSTALL_RULE
+        mkdir -p $(bindir)
 	$(INSTALL) -m 755 $(NVIDIA_SETTINGS) $(bindir)/$(NVIDIA_SETTINGS)
 	mkdir -p $(mandir)
 	$(INSTALL) -m 644 doc/$(MANPAGE) $(mandir)
-	gzip -9f $(mandir)/$(MANPAGE)
 endef
 endif
 
@@ -118,7 +118,7 @@ X11_CFLAGS = $(X11_INC_DIRS)
 GTK_CFLAGS := $(shell $(PKG_CONFIG) --cflags gtk+-2.0)
 GTK_LDFLAGS := $(shell $(PKG_CONFIG) --libs gtk+-2.0)
 
-X11_LIBS := $(X11_LIB_DIRS) -Wl,-Bstatic -lXxf86vm -Wl,-Bdynamic -lX11 -lXext
+X11_LIBS := $(X11_LIB_DIRS) -Wl,-Bdynamic -lX11 -lXext -lXxf86vm
 
 XNVCTRL_LIB := src/libXNVCtrl/libXNVCtrl.a
 XNVCTRL_DIR := src/libXNVCtrl
@@ -195,7 +195,6 @@ DEPS := $(patsubst %.c,$(DEPS_DIR)/%.d,$
 all: $(NVIDIA_SETTINGS) doc/$(MANPAGE)
 
 install: all
-	$(STRIP) $(NVIDIA_SETTINGS)
 	$(INSTALL_RULE)
 
 $(OBJS_DIR)/%.o: %.c
